### DISCUSSION TITLE
Coral Dev board mini

### DISCUSSION THREAD

**Discussion by joelegg** [blakeblackshear/frigate#3850] [Ask A Question]
hi all -  need some advice please - I bought a coral usb accelerator from ebay  but received a dev board mini like this: https://au.element14.com/coral/g650-03324-01/development-board-mini-soc/dp/3885864?gclid=Cj0KCQjwmouZBhDSARIsALYcoup6ZhsnjuWnbz411UJIgwbzku8w2JzvD0kH2EGAu0_iHLQCZfdL70waAkOqEALw_wcB&mckv=_dc|pcrid||pkw||pmt||slid||product|3885864|pgrid||ptaid||&CMP=KNC-GAU-GEN-SHOPPING-PLA-PMAX

I have a synology 1019+ - can i connect this to a usb cable and run it on my synology? I am not sure if i have been ripped off as i thought i was getting something like this:  https://www.ebay.com.au/itm/195187240322?chn=ps&_ul=AU&norover=1&mkevt=1&mkrid=705-139619-5960-0&mkcid=2&itemid=195187240322&targetid=&device=c&mktype=pla&googleloc=9071347&poi=&campaignid=15791083372&mkgroupid=&rlsatarget=&abcId=9300816&merchantid=116294652&gclid=Cj0KCQjwmouZBhDSARIsALYcouoZ_S9OH4wjqHkai7HoOOw5vXzVPeg8CHBiU2nek_FMv4heeRF_lIUaAvgVEALw_wcB

thank you

**Comment by NickM-27** [comment]
No, to run frigate officially would need to run on the dev board mini. 

It's not official but another user has been investigating use the coral dev board remotely with decent results, check out https://github.com/blakeblackshear/frigate/issues/3449

**Comment by joelegg** [comment]
Thanks so much. I got a partial refund and will try sell the dev board. Ordered a USB one from different seller. If anyone wants the dev board let me know
